src/ConfigSetup.cpp

```diff
@@ -1704,6 +1723,11 @@ void ConfigSetup::verifyInputs(void)
     printf("Warning: Cell dimension set, but will be ignored in restart mode.\n");
   }
 
+  if (in.restart.restartFromCheckpoint && !in.restart.enable){
+    std::cout << "Error: Checkpoint cannot be used without Restart true!" << std::endl;
+    exit(EXIT_FAILURE);    
+  }
+
```

Comment:
Why are we still using `in.restart.enable`?

Reply:
Because the only way to know whether a box of orthogonal/nonorthogonal before the XSC simulation is loaded is to either read the Config file or the top of the PDB.  

That variable in.restart.enable ensures the REMARK with the PDB cryst is read at the top of the PDB.  Since we want to allow the user to comment out the Cell Basis in a restart.

An alternative would remove the "Restart true" from the conf file and to set 
in.restart.enable = (restartFromXSC && restartFromCoor).